experimenter/experimenter/experiments/models.py

```diff
@@ -837,6 +837,25 @@ def excluded_live_deliveries(self):
             )
         return matching
 
+    @property
+    def live_experiments_in_namespace(self):
+        matching = []
```

Comment:
nit: you can remove this since we're not branching anymore

Reply:
🫡